### PR TITLE
Add state to ECS services. Allow deleting a service.

### PR DESCRIPTION
**Pull request by mikhail**
@swaggy @diranged

### CODE REVIEW

File: kingpin/actors/aws/base.py [hunk 1]
```diff
@@ -210,6 +211,9 @@ def thread(self, function, *args, **kwargs):
                 raise exceptions.InvalidCredentials(msg)
 
             raise
+        except boto3_exceptions.Boto3Error as e:
```
Comment (diranged) [inline kingpin/actors/aws/base.py]: Can you talk about what you're doing here?

Comment (diranged) [inline kingpin/actors/aws/base.py]: Nevermind -- I was mistaking this for our _actual base actor class_..

**Comment by diranged** [comment]
Looks fine to me.. @swaggy should take a peek since he's more familiar with the ECS API tho.

File: kingpin/actors/aws/base.py [hunk 2]
```diff
@@ -210,6 +211,9 @@ def thread(self, function, *args, **kwargs):
...
+            raise exceptions.RecoverableActorFailure(
+                'Boto3 had a failure: %s' % e)
```
Comment (swaggy) [inline kingpin/actors/aws/base.py]: Sweet

File: kingpin/actors/aws/ecs.py [hunk 3]
```diff
@@ -543,6 +540,9 @@ class Service(ECSBaseActor):
 
     **Options**
 
+    :state:
+      Present or Absent. Defaults to Present.
```
Comment (swaggy) [inline kingpin/actors/aws/ecs.py]: Use the same string here as below, "Desired state: present/absent. Defaults to present."

**Comment by swaggy** [comment]
LGTM 🎆

**Pull request merged**